internal/driver/html/stacks.js

```diff
@@ -606,3 +574,40 @@ function stackViewer(stacks, nodes) {
     return hsl;
   }
 }
+
+// Mapping from unit to a list of display scales/labels.
+// List should be ordered by increasing unit size.
+const pprofUnits = new Map([
```

Comment:
I wish we switch to the unit definitions in google3/third_party/golang/pprof/internal/measurement/measurement.go (towards the end of that file is the table).